alchemist-loading/src/test/resources/testExporters.yml

```diff
@@ -7,3 +7,11 @@ export:
     data:
       - time
       - molecule: "default_module:default_program"
+
+  - type: MongoDBExporter
+    parameters:
+      uri: "mongodb+srv://alchemist-user:alchemist-pwd@alchemist.ewxds.mongodb.net/myFirstDatabase?retryWrites=true&w=majority"
```

Comment:
this is fragile. Extremely fragile. And this is a password in clear, or am I wrong?

Reply:
Done: changed to a local instance of mongo.